masterlist.yaml

```
@@ -22646,3 +22646,11 @@ plugins:
     clean:
       - crc: 0xBD12E363
         util: 'SSEEdit v4.0.4'
+
+  - name: 'WorldEaterBeater.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/60778' ]
+    after: [ 'DeadlyDragons.esp' ]
+    inc: [ 'Dragon Combat Overhaul.esp' ]
+    clean:
+      - crc: 0xAE2F7427
+        util: 'SSEEdit v4.0.4'
```

There should be a newline at the end of the masterlist.

```suggestion
        util: 'SSEEdit v4.0.4'

```